pkgs/by-name/me/menulibre/package.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, gnome-menus
+, gtk3
+, intltool
+, gobject-introspection
+, wrapGAppsHook
+, testers
+, menulibre
+}:
+
+python3Packages.buildPythonPackage rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```

sorry, hopefully this is last change